src/technologies/j.json

```diff
@@ -268,9 +268,17 @@
     "cpe": "cpe:2.3:a:jenkins:jenkins:*:*:*:*:*:*:*:*",
     "description": "Jenkins is an open-source automation tool written in Java with plugins built for Continuous Integration (CI) purposes.",
     "headers": {
-      "X-Jenkins": "([\\d.]+)\\;version:\\1"
+      "X-Jenkins": "([\\d.]+)\\;version:\\1",
+      "X-Hudson": "",
+      "X-Hudson-Theme": "",
+      "X-Hudson-CLI-Port": "",
+      "X-Jenkins-CLI-Port": "",
+      "X-Instance-Identity": ""
     },
-    "html": "<span class=\"jenkins_ver\"><a href=\"https://jenkins\\.io/\">Jenkins ver\\. ([\\d.]+)\\;version:\\1",
+    "html": [
+      "<span class=\"jenkins_ver\"><a href=\"https://jenkins\\.io/\">Jenkins ver\\. ([\\d.]+)\\;version:\\1",
```

Comment:
I think, wappalyzer doesn't explicitly have a priority for `;version` based filters. If there are more (like in this case, where there is one in X-Jenkins header and one in HTML), both match, and they are different, I don't know which one will be taken.

This is not necessarily a problem, since usually the versions from multiple sources should be the same.